s3torchbenchmarking/src/s3torchbenchmarking/dataset/benchmark.py

```diff
@@ -43,38 +40,23 @@ def run_experiment(config: DictConfig):
         batch_size=config.dataloader.batch_size,
     )
 
-    result = model.train(dataloader, config.training.max_epochs)
-    root_config = HydraConfig.get()
-    output_dir = root_config.runtime.output_dir
-    job_result_path = write_result(result, Path(output_dir))
-    print(f"{root_config.job.name} results written to: {job_result_path}")
-
-
-def write_result(result: ExperimentResult, out_dir: Path) -> Path:
-    result_path = out_dir / "result.json"
-    with open(result_path, "w") as outfile:
-        json.dump(result, outfile, cls=ExperimentResultJsonEncoder)
+    result = model.train(dataloader, config.epochs)
 
-    return result_path
+    metrics = {
+        "throughput_mibs": [result.volume / result.elapsed_time],
```

Comment:
Can we preserve information about elapsed_time too? Sometimes it is more convenient to have time, rather than throughput.

Reply:
Sure, lemme add it